Makefile.third

```diff
@@ -317,7 +317,20 @@ ifdef ANDROID
 	readelf -d $@ | grep "no dynamic" \
 		|| echo "warning: Android L+ support only PIE binary and can't run this"
 endif
+# ===========================================================================
+# dropbear
 
+$(OUTPUT_DIR)/dropbear: $(ZLIB) $(THIRDPARTY_DIR)/dropbear/CMakeLists.txt
```

Comment:
@poire-z I'll check on my computer in a bit but I assume the problem is this here. No `dropbear` file is ever created, only `dropbearmulti`.